### PR TITLE
Chaining on and off

### PR DESCRIPTION
**Pull request by tanhauhau**
- Generated .min.js
- Updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +99,13 @@ Install via [npm](https://npmjs.org/package/fishbone):
 npm install fishbone
 ```
 
+Intall via bower:
```
Comment (aemkei) [inline README.md]: Small typo. Should be: "Install"

File: README.md [hunk 2]
```diff
@@ -100,6 +99,13 @@ Install via [npm](https://npmjs.org/package/fishbone):
...
+
+```sh
+bower install --save https://github.com/tanhauhau/fishbone.js.git
```
Comment (aemkei) [inline README.md]: Would be great to link to the original repo at: 
https://github.com/aemkei/fishbone.js.git

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "description": "A super lightweight (½kb) JavaScript library with automatic method chaining, automatic context binding, event support and simple inheritance.",
+  "main": "fishbone.min.js",
+  "authors": [
+    "Martin Kleppe <kleppe@ubilabs.net>"
+  ],
+  "license": "WTFPL",
+  "keywords": [
+    "Javascript"
+  ],
+  "homepage": "https://github.com/tanhauhau/fishbone.js",
```
Comment (aemkei) [inline bower.json]: Would you mind changing this to aemkei, too?

**Comment by aemkei** [comment]
This is great! Just added some small notes and then it is ready to merge.

**Comment by tanhauhau** [comment]
Updated with a new commit

**Comment by aemkei** [comment]
Thanks again! Merged and looks good.

**Comment by aemkei** [comment]
PS: I added it to the Bower registry. So `bower install fishbone` should work.

**Pull request merged**